site_root/source/resources/StructureDefinition-Phenotype.json

```diff
@@ -0,0 +1,36 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "Phenotype",
+  "url": "http://fhir.kids-first.io/StructureDefinition/Phenotype",
+  "name": "Phenotype",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
```

Comment:
Does the latest publisher still work with this version? I remember version 4.0.0 disappeared when 4.0.1 came out.

Reply:
I believe so since everything is passing validation. Do you mean the latest publisher with the new IG structure? I'm not sure, but I think this is something that will be easy to mass update later when we switch over to the new IG structure.